### ISSUE TITLE
Error while loading "calib_cam_to_cam.txt" - can not reshape the array.

### ISSUE THREAD

**Issue by gautamjain1009** [fangchangma/self-supervised-depth-completion#34]
I am using your pre-trained models for testing on the validation set "val_selection_cropped". But while loading the **"calib_cam_to_cam.txt"**  certain errors come. I attaching the screenshot of the error. 

![error_calib_to_cam](https://user-images.githubusercontent.com/37597326/69817323-a078f800-11fa-11ea-944e-e158b9290626.png)

I am running the script with this command in the terminal: - 

python3 /scratch/gjain2s/Approaches/sparse_dense/self-supervised-depth-completion/main.py --data-folder /scratch/gjain2s/Approaches/sparse_dense/data  --evaluate /home/gjain2s/self_supervised/model_best.pth.tar --val select 

Any help would be highly appreciated.
